plugin/build_baron.go

```diff
@@ -34,10 +34,11 @@ func (bbp *BuildBaronPlugin) Configure(conf map[string]interface{}) error {
 	}
 
 	for projName, proj := range bbpOptions.Projects {
-		if proj.TicketCreateProject == "" {
-			return fmt.Errorf("ticket_create_project cannot be blank")
+		webhookConfigured := proj.TaskAnnotationSettings.FileTicketWebHook.Endpoint != ""
```

Comment:
It makes me a bit wary that we have so many errors here that prevent the app servers from coming up. Does it seem worth making a separate ticket to make this less aggressive? Or do we really not want the app server to come up if these things aren't configured correctly?

Reply:
I'm not sure, that would probably be a question for @john-m-liu.

Reply:
I think you can make that determination. If you'd like errors here to not prevent the app from starting, you can log them rather than returning them

Reply:
I switched line 65 to logging instead of an error.